project/Build.scala

```diff
@@ -1506,15 +1508,24 @@ object Build {
               (`scala3-interfaces`/Compile/products).value,
               (`tasty-core-bootstrapped`/Compile/products).value,
               (`scala3-library-bootstrapped`/Compile/products).value,
-              // TODO we can't load stdlib from Tasty
```

Comment:
IMHO we should leave this here until we will be able to generate stdlib togeter with rest of Scala3 jars

Reply:
We have a similar comment above, where we define `generateScala3StdlibDocumentation`.